d3a_api_client/redis_market.py

```diff
@@ -58,7 +59,7 @@ def list_market_stats(self, market_slot_list):
         self.redis_db.publish(f"{self.area_id}/market_stats", json.dumps({"market_slots": market_slot_list}))
         return self._wait_and_consume_command_response("market_stats")
 
-    def change_grid_fees_const(self, fee_cents_kwh):
+    def grid_fees(self, fee_cents_kwh):
```

Comment:
Was this a requirement from Colin? IMO `grid_fees` does not tell mw what the function does.

Reply:
Sorry, just read the story description. But IMO `get_grid_fees` would be more descriptive.

Reply:
Yes your suggestion makes sense but then we had to rename it in `RestMarket` too here(https://github.com/gridsingularity/d3a-api-client/blob/master/d3a_api_client/rest_market.py#L37) which I don't want to disturb that is possibly already been in use by some external data scientists

Reply:
Ahh, it is actually SETTING the grid fees. Ok. 
Yeah, keep it as `grid_fees`.